apps/flows/src/app/transaction-details-flow/for-you/for-you.component.html

```diff
@@ -1,26 +1,21 @@
 <kirby-page title="For you" defaultBackHref="/">
   <kirby-page-content>
-    <div class="app-header"></div>
     <div class="container">
-      <kirby-card class="spacing" [hasPadding]="true"
-        ><h2>Credits in account</h2>
-        <h1>2.000.000 DKK</h1>
+      <kirby-card [hasPadding]="true"
+        ><h1>Credits in account</h1>
+        <h2>2.000.000 DKK</h2>
```

Comment:
This should probably not be a heading at all, but since this is "just" a user flow example I think it's ok to leave it for now.